bundles/org.openhab.binding.ism8/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -124,6 +128,10 @@
 	<channel-type id="temperature-r">
```

Comment:
the script missed `number-r`, `percentage-r`, `percentage-rw`....
the `-r` should be `Measurement` (or Status).....?

Reply:
Yes the -r and -rw suffixes give extra information..